numpy/lib/tests/test_packbits.py

```diff
@@ -213,6 +213,15 @@ def test_packbits_large():
     assert_raises(TypeError, np.packbits, np.array(a, dtype=float))
 
 
+def test_packbits_very_large():
+    # test some with a larger arrays gh-8637
+    # code is covered earlier but larger array makes crash on bug more likely
```

Comment:
The test is a big excessive but the full suite still runs in 0.3 seconds which is fine for fast. But we need to recategorize the tests a bit, a few very slow tests are run by default and probably don't need to anymore.